simple-markdown.js

```diff
@@ -357,13 +352,14 @@ var parserFor = function(rules /*: ParserRules */, defaultState /*: ?State */) {
 
             do {
                 var currOrder = currRule.order;
-                var currCapture = currRule.match(source, state, prevCapture);
+                var prevCaptureStr = state.prevCapture === null ? "" : state.prevCapture[0];
```

Comment:
If we wanted to maintain the legacy behavior here we could add an `|| i === 0` to the condition, which would reset it at the beginning of each nesting like it used to.

Reply:
Makes sense. I think I'll do that so we don't break any legacy code, but will encourage `state.prevCapture` in the future.